ostelco-ios-client/Coordinators/OnboardingCoordinator.swift

```diff
@@ -303,6 +303,13 @@ extension OnboardingCoordinator: MyInfoSummaryDelegate {
     func verifiedSingPassAddress() {
         advance()
     }
+
+    func failedToLoadMyInfo() {
+        if localContext.selectedVerificationOption == .singpass {
```

Comment:
I don't think there's a need for the conditional here.

Reply:
done